src/backend.md

```diff
@@ -18,6 +19,236 @@
 Table of contents can be generated in services like http://ecotrust-canada.github.io/markdown-toc/
 -->
 
+## Project structure
+
+Project structure should follow next principles:
+
+1. Not needed to move files without changing their purpose.
```

Comment:
What does it mean?

Reply:
I took it from frontend section. Let's remove it.